package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "js-dataverse",
-  "version": "1.0.48",
-  "description": "A Dataverse module for JavaScript/TypeScript",
+  "version": "2.0.0",
```

Comment:
I'm not sure if I would push non-stable builds to npm. It seems safer to me to use local versions for the moment to test the package (it may even be faster). We can talk more about this in the frontend sync meeting.

Reply:
I guess the version doesn't matter so much until we actual publish to npm. We can talk about it later and change the version later, if need be.